### PR TITLE
 Added "activePage" as a active class in navigation menu 

### PR DESCRIPTION
**Pull request by taruntds**
activePage is a active class which will be applied when any page is active. For example. If you visit any category page then this "activePage" class will be activated and that particular category name will be highlighted in different color.

### CODE REVIEW

**Comment by bigmadness** [comment]
active for pages?

**Comment by taruntds** [comment]
Hello @bigmadness 

For web pages, i am working on it and soon i will submit that too to cornerstone team.

**Comment by taruntds** [comment]
@bigmadness 

I have added the activePage class too for web pages. Please check.

**Comment by taruntds** [comment]
@karen-white @junedkazi 

can anyone please let me know whether my changes are merged in cornerstone or not. Can i close this pull request if it is merged or not.

**Comment by karen-white** [comment]
@taruntds Thanks for submitting this! Our team will review the changes as soon as they have a moment. Once it's reviewed, we'll comment if there are any questions, or merge it, in which case the pull request would close automatically.

File: assets/scss/components/stencil/navPages/_navPages.scss [hunk 1]
```diff
@@ -209,7 +209,7 @@
         }
     }
 
-    &:hover {
+    &:hover, &.activePage {
```
Comment (junedkazi) [inline assets/scss/components/stencil/navPages/_navPages.scss]: can we just use `active` as a class name so it is uniform across the codebase. We can always refer to it in a nested way for css.

Comment (junedkazi) [reply assets/scss/components/stencil/navPages/_navPages.scss]: you can always provide specific override for nav only using something like `.nav .active`

**Comment by Ubersmake** [comment]
Commits 31365c7a235c190d2c7d8addf755c330240de08f and 643272619f14c4f48a455d8efd9ef5c2ccc014cc were squashed and pushed through with #1354

**Pull request closed without merging**